src/mgr/ActivePyModules.cc

```diff
@@ -671,7 +671,6 @@ PyObject *ActivePyModules::get_typed_config(
   } else {
     dout(10) << " " << key << " not found " << dendl;
   }
-  with_gil_t with_gil{no_gil};
```

Comment:
shall we acquire the GIL in this branch as well? i.e., to trade the `with_gil_t with_gil{no_gil}` for a `no_gil.acquire_gil()` in this branch.

Reply:
I think we don't need it since Py_RETURN_NONE is a constant, so no code executes before ``~without_gil_t()`` retakes the lock